frontend/src/plugins/demo/index.ts

```diff
@@ -41,38 +28,28 @@ window.addEventListener(
       const demoName = serverInfo.demoName;
       if (demoName) {
         storage.set({
-          hint: {
+          demo: {
             name: demoName,
           },
         });
-        hideConsoleGuides();
-        initHintListeners();
-      }
-    } else {
-      // This is using for dev mainly.
-      const { guide: storageGuide, hint: storageHint } = storage.get([
-        "guide",
-        "hint",
-      ]);
-      const params = new URLSearchParams(window.location.search);
-      const paramGuide = params.get("guide");
-      const paramHint = params.get("hint");
 
-      if (paramGuide || storageGuide) {
-        hideConsoleGuides();
-        initGuideListeners();
-      }
-      if (paramHint || storageHint) {
-        hideConsoleGuides();
-        initHintListeners();
+        // mount the demo vue app
+        const demoDrawerContainer = document.createElement("div");
+        document.body.appendChild(demoDrawerContainer);
+        const app = createApp(DemoWrapper);
+        app.use(router);
+        app.mount(demoDrawerContainer);
```

Comment:
I try to mount another vue app into body to render some logical component for demo. Can I do that in this way? @LiuJi-Jim

Reply:
And I'd like to rewrite all pure js elements into vue component. wdyt?